<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMenu</class>
 <widget class="QWidget" name="FormMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormMenu</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>690</y>
     <width>1024</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/home.png</normaloff>:/rc/icon/home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QStackedWidget" name="page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>640</height>
    </rect>
   </property>
   <widget class="FormMenuPList" name="page_PList"/>
   <widget class="FormMenuDebug" name="page_Debug"/>
   <widget class="FormMenuPSet" name="page_PSet"/>
   <widget class="FormMenuHSet" name="page_HSet"/>
   <widget class="FormMenuIO" name="page_IO"/>
   <widget class="FormMenuCalib" name="page_Calib"/>
   <widget class="FormMenuMac" name="page_Mac"/>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>1024</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>程序列表</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>手动调试</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>程序设置</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>高级设置</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>输入输出</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>校准</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>本机</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>700</y>
     <width>584</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">.QWidget {
border: 1px solid white;
border-radius: 10px;
}
QLabel {
font-size: 18px;
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="prm_id">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="prm_name">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>产品名称：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>程序号：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormMenuPList</class>
   <extends>QWidget</extends>
   <header>FormMenuPList.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuPSet</class>
   <extends>QWidget</extends>
   <header>FormMenuPSet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuDebug</class>
   <extends>QWidget</extends>
   <header>FormMenuDebug.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuIO</class>
   <extends>QWidget</extends>
   <header>FormMenuIO.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuHSet</class>
   <extends>QWidget</extends>
   <header>FormMenuHSet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuCalib</class>
   <extends>QWidget</extends>
   <header>FormMenuCalib.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuMac</class>
   <extends>QWidget</extends>
   <header>FormMenuMac.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
